App/Data/DataSource/Auth/AuthManager.swift

```diff
@@ -11,6 +11,7 @@ class AuthManager {
     private let urlRequestUtils = NetworkUtilities.shared
     private let keychainManager = KeyChainManager()
     private let urlSession: URLSession = URLSession.shared
+    private let networkController: NetworkController = .shared
```

Comment:
all three lines 11,13,14 have different initialization and typing 

maybe we can make it the same across the file/project later
private let variable: Class = .shared